cpp/src/stream_compaction/stream_compaction.cu

```diff
@@ -108,4 +112,78 @@ gdf_column drop_nulls(gdf_column const &input) {
   }
 }
 
+rmm::device_vector<gdf_index_type>
+get_unique_ordered_indices(const cudf::table& key_columns,
```

Comment:
In case you don't make this a public API, you should probably put it in `detail` namespace or the anonymous namespace right above this.